suppressions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shouldn't we wait for gRPC update and remove this suppression?

Reply:
This suppression related to grpc:grpc library with one isn't used by us.
Looks like it is a CPE converter problem, because most of library has the same mark.